db/seeds.rb

```diff
@@ -5,70 +5,95 @@
 require 'paper_trail'
 require 'timecop'
 
-include FactoryBot::Syntax::Methods
+module Seeds
+  extend FactoryBot::Syntax::Methods
```

Comment:
Use `extend` instead of `include` [because we are in a module now](https://www.geeksforgeeks.org/include-v-s-extend-in-ruby/).